pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -674,19 +674,6 @@ func (m *kubeGenericRuntimeManager) killContainer(pod *v1.Pod, containerID kubec
 		gracePeriod = *pod.DeletionGracePeriodSeconds
 	case pod.Spec.TerminationGracePeriodSeconds != nil:
 		gracePeriod = *pod.Spec.TerminationGracePeriodSeconds
-
-		if utilfeature.DefaultFeatureGate.Enabled(features.ProbeTerminationGracePeriod) {
```

Comment:
The feature gate should be removed from the kubelet, but we still need to keep the logic inside.

Reply:
Would it be sufficient to comment out this `if` block as it would allow for the feature to remain in the code but not be utilized?

Reply:
No, we shouldn't be leaving comments. The feature gate should be removed.

Reply:
if this feature is still beta and you remove the feature gate check, 
what happens if someone wants to disable all beta features or this specific feature?